src/coreclr/src/jit/compiler.h

```diff
@@ -5541,7 +5542,7 @@ class Compiler
     //                      or BB_UNITY_WEIGHT when we aren't using profile data.
     unsigned fgProfileRunsCount()
     {
-        return fgIsUsingProfileWeights() ? fgNumProfileRuns : BB_UNITY_WEIGHT;
+        return fgIsUsingProfileWeights() ? fgNumProfileRuns : BB_UNITY_WEIGHT_UNSIGNED;
```

Comment:
Update function header comment to match. IMO, the function semantics seem weird.

Reply:
Yes, this is an odd one; it is really trying to guess global importance.

Reply:
Fixed.